tests/helpers/testHelper/testHelper.ts

```
@@ -0,0 +1,40 @@
+import mongoose from "mongoose";
+import { MongoMemoryServer } from "mongodb-memory-server";
+
+export class TestHelper {
+  private static _mongod: MongoMemoryServer;
+
+  // Starts the in-memory database
+  static async startDatabase(): Promise<void> {
+    this._mongod = await MongoMemoryServer.create();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)